holidays/countries/india.py

```diff
@@ -28,269 +35,383 @@ class India(HolidayBase, ChristianHolidays, InternationalHolidays, IslamicHolida
+
+        # Mahavir Jayanti.
+        self._add_mahavir_jayanti(tr("Mahavir Jayanti"))
+
+        # Maha Shivaratri.
+        self._add_maha_shivaratri(tr("Maha Shivaratri"))
+
+        # Navratri / Sharad Navratri.
+        self._add_sharad_navratri(tr("Navratri / Sharad Navratri"))
+
+        # Ram Navami.
+        self._add_ram_navami(tr("Ram Navami"))
+
+        # Maha Navami.
+        self._add_maha_navami(tr("Maha Navami"))
+
+        # Ganesh Chaturthi.
+        self._add_ganesh_chaturthi(tr("Ganesh Chaturthi"))
+
+        # Makar Sankranti.
+        self._add_makar_sankranti(tr("Makar Sankranti"))
+
+        # Guru Nanak Jayanti.
+        self._add_guru_nanak_jayanti(tr("Guru Nanak Jayanti"))
 
         # Islamic holidays.
         # Day of Ashura.
-        self._add_ashura_day("Day of Ashura")
+        self._add_ashura_day(tr("Day of Ashura"))
 
         # Birth of the Prophet.
-        self._add_mawlid_day("Mawlid")
+        self._add_mawlid_day(tr("Mawlid"))
 
         # Eid ul-Fitr.
-        name = "Eid ul-Fitr"
+        name = tr("Eid ul-Fitr")
         self._add_eid_al_fitr_day(name)
         self._add_eid_al_fitr_day_two(name)
 
         # Eid al-Adha.
-        name = "Eid al-Adha"
+        name = tr("Eid al-Adha")
         self._add_eid_al_adha_day(name)
         self._add_eid_al_adha_day_two(name)
 
         # Christian holidays.
-        self._add_palm_sunday("Palm Sunday")
-        self._add_good_friday("Good Friday")
-        self._add_easter_sunday("Easter Sunday")
-        self._add_whit_sunday("Feast of Pentecost")
-        self._add_christmas_day("Christmas Day")
+        self._add_palm_sunday(tr("Palm Sunday"))
+        self._add_good_friday(tr("Good Friday"))
+        self._add_easter_sunday(tr("Easter Sunday"))
+        self._add_whit_sunday(tr("Feast of Pentecost"))
+        self._add_christmas_day(tr("Christmas Day"))
 
         if self.subdiv == "OR":
             self._populate_subdiv_od_public_holidays()
 
+    # Andaman and Nicobar Islands.
+    def _populate_subdiv_an_public_holidays(self):
+        self._add_holiday_apr_14(tr("Dr. B. R. Ambedkar's Jayanti"))
+
     # Andhra Pradesh.
     def _populate_subdiv_ap_public_holidays(self):
-        self._add_holiday_apr_14("Dr. B. R. Ambedkar's Jayanti")
-        self._add_holiday_nov_1("Andhra Pradesh Foundation Day")
+        self._add_holiday_apr_14(tr("Dr. B. R. Ambedkar's Jayanti"))
+        self._add_holiday_nov_1(tr("Andhra Pradesh Foundation Day"))
 
     # Assam.
     def _populate_subdiv_as_public_holidays(self):
-        self._add_holiday_apr_15("Bihu (Assamese New Year)")
+        self._add_makar_sankranti(tr("Bihu"))
+        self._add_holiday_dec_2(tr("Asssam Day"))
 
     # Bihar.
     def _populate_subdiv_br_public_holidays(self):
-        self._add_holiday_mar_22("Bihar Day")
-        self._add_holiday_apr_14("Dr. B. R. Ambedkar's Jayanti")
+        name = tr("Chhath Puja")
+        self._add_chhath_puja(name)
```

Comment:
```suggestion
        self._add_chhath_puja(tr("Chhath Puja"))
```
And in similar cases below.